2023 and before 1 changes

Bonnarel opened the pull request:
Add transform parameters, pixel ranges, Dataset format changes, product type reduction

Bonnarel committed:
Update SODA.tex

RElated to issue #4 this Commit introduces a couple of subsections for new rebinning / reprojecting parameters in SODA

Bonnarel committed:
Merge branch 'ivoa-std:master' into Rebin-Reproject-Params

Commit:
reintroduce WCS parameters

Commit:
Add FORMAT DPTYPE PIXELS METADATA

pdowler reviewed (commented):
Some organisational suggestions:
- change DOCTYPE back to WD in the Makefile
- re-order the filtering param sections so that MOC comes after POLYGON and PIXELS comes at the end
- isn't DPTYPE a kind of transformation (in the general sense)? Move it to the "Transform" section?

I will comment on individual items separately.

Bonnarel committed:
Update SODA.tex

adding sub-sampling by picking up "1 upon n" pixels in the area of interest

Bonnarel committed:
Update Makefile

Bonnarel committed:
Update Makefile

Bonnarel committed:
Update SODA.tex

moving DPTYPE subsection to Transform and reorder the filtering section according to Pat's comments

Bonnarel commented:
> Some organisational suggestions:
> 
>     * change DOCTYPE back to WD in the Makefile
> 
>     * re-order the filtering param sections so that MOC comes after POLYGON and PIXELS comes at the end
> 
>     * isn't DPTYPE a kind of transformation (in the general sense)? Move it to the "Transform" section?
> 
> 
> I will comment on individual items separately.

Changes made

Bonnarel committed:
Update Makefile

Working draft 1.1 first version

Bonnarel committed:
Update SODA.tex

clarifying behavior for DPTYPE parameter

Bonnarel committed:
Update SODA.tex

- PIXELS and parameter multiplicity
- RESPONSEFORMAT = application/hips specificities
